tiledb/highlevel.py

```diff
@@ -120,6 +120,54 @@ def array_fragments(uri, ctx=None):
     return tiledb.FragmentInfoList(uri, ctx)
 
 
+def delete_fragments(
+    uri, timestamp_range, config=None, ctx=None, verbose=False, dry_run=False
+):
+    """
+    Delete fragments from an array located at uri that fall within a given
+    timestamp_range.
+
+    :param str uri: URI for the TileDB array (any supported TileDB URI)
+    :param config: Override the context configuration. Defaults to None,
+        inheriting the context parameters.
+    :param (int, int) timestamp: (default None) If not None, vacuum the array
+        using the given range (inclusive)
+    :param ctx: (optional) TileDB Ctx
+    :param verbose: (optional) Print fragments being deleted (default: False)
+    :param dry_run: (optional) Preview fragments to be deleted without
+        running (default: False)
+    :return: FragmentsInfo object
+    """
+
+    if not isinstance(timestamp_range, tuple) and len(timestamp_range) != 2:
+        raise TypeError(
+            "'timestamp_range' argument expects tuple(start: int, end: int)"
+        )
+
+    if verbose or dry_run:
+        print("Deleting fragments...")
+
+    if not ctx:
+        ctx = tiledb.default_ctx()
+
+    if config is None:
+        config = tiledb.Config()
```

Comment:
If the config is empty, should we set it to the ctx's config? Or leave it as none for vfs?

Reply:
Thanks for catching this. You're right it should be `ctx.config()`.